test/index.test.ts

```diff
@@ -51,6 +52,8 @@ const ALLOWED_FEATURE_NAMES = [
   'xrSpatialTracking',
 ];
 
+const DASHED_FEATURE_NAMES = ALLOWED_FEATURE_NAMES.map((feature) => decamelize(feature, '-'));
```

Comment:
Could you instead add this manually to the source code (and uninstall `decamelize`)? I know it's repetitive, but I want to avoid adding anything fancy to my tests.

Reply:
Of course.